utils/node_local_test.py

```diff
@@ -396,8 +396,9 @@ def start(self):
 
             if ready:
                 break
-            if time.time() - start > 10:
+            if time.time() - start > 20:
                 raise Exception("Failed to start")
+        print('Server started in {:.2f} seconds'.format(time.time() - start))
```

Comment:
Small efficiency nit:

```suggestion
            end = time.time()
            if end - start > 20:
                raise Exception("Failed to start")
        print('Server started in {:.2f} seconds'.format(end - start))
```

Reply:
I'm slightly worried that python will warn about possibly undefined loop variable use if I do that.